client/src/components/auth/LoginPatient.component.js

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment, useState } from 'react';
+import { Link, Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { login } from '../../actions/auth';
+
+const LoginPatient = ({ login, isAuthenticated }) => {
+  const [formData, setFormData] = useState({
+    email: '',
+    password: '',
+  });
+
+  const { email, password } = formData;
+
+  const onChange = (e) =>
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+
+  const onSubmit = async (e) => {
+    e.preventDefault();
+    login(email, password);
+  };
+
+  if (isAuthenticated) {
+    return <Redirect to='/patient/login/:id' />;
```

Comment:
In the future we would search the email from the patient DB, retrieve their ID and redirect them to /patient/login/{actual id}???

Reply:
I did not really update much of the logic, from the last approach.  We can use this a sort of jump off point.